### ISSUE TITLE
Error retrieving JSON attribute (rvals[j].trim)

### ISSUE THREAD

**Issue by lucasrolim-intuitive** [casbin/node-casbin#31]
I'm trying to use an ABAC policy model, but I'm having some problems to retrieve the attributes from my JSON variable. Here is a simplified example:

My policy file:

```csv
p, alice, data1, read
```

Model file:

```ini
[request_definition]
r = user, resource, action

[policy_definition]
p = user, resource, action

[policy_effect]
e = some(where (p.eft == allow))

[matchers]
m = r.user == p.user && r.resource.name == p.resource && r.action == p.action
```

JS call

```js
// req.body.resource = { name: 'data1' }
// req.body.action = read
enforcer.enforce("alice", req.body.resource, req.body.action)
```

This returns: 

`UnhandledPromiseRejectionWarning: TypeError: rvals[j].trim is not a function`

It works fine when I set  'data1'  as enforcer's second argument and remove the ".name" from the matcher.

I have  already read [TypeError: rvals[j].trim is not a function](https://github.com/casbin/node-casbin/issues/27), but don't found a solution.

**Comment by hsluoyz** [comment]
@lucasrolim-intuitive can you try if this issue is fixed by the latest commit? You may need to copy and paste the source code. When you conform it's fixed, we will release a new version in NPM.

**Comment by lucasrolim-intuitive** [comment]
It looks ok now. Thank you for the quick answer!

**Comment by hsluoyz** [comment]
@lucasrolim-intuitive we have released v1.1.6, which has this issue fixed: https://www.npmjs.com/package/casbin/v/1.1.6
